src/components/Navbar/index.js

```diff
@@ -1,58 +1,47 @@
-import React, { Component } from "react";
+import React, { useState } from "react";
 import PropTypes from "prop-types";
 import { capitalize } from "lodash-es";
-
 import Section from "root/components/Section";
 import Typography from "root/components/Typography";
 import Logo from "root/components/Logo";
 import Link from "root/components/Link";
+import Wave from "root/assets/images/wave-mobile-menu-background.inline.svg";
+import classNames from "classnames";
 
 import "./index.css";
 
-const COLOR_THEME_PRIMARY = {
-  color: "klein-blue",
-  underlineColor: "klein-blue",
-  logoColor: "klein-blue",
-};
-
-const COLOR_THEME_SECONDARY = {
-  color: "baby-blue",
-  underlineColor: "baby-blue",
-  logoColor: "white",
-};
+const Navbar = ({ currentPage, theme }) => {
+  const colors = () => {
+    const COLOR_THEME_PRIMARY = {
+      color: "klein-blue",
+      underlineColor: "klein-blue",
+      logoColor: "klein-blue",
+    };
 
-export default class Navbar extends Component {
-  static propTypes = {
-    currentPage: PropTypes.oneOf(["home", "company", "blog", "careers"]),
-    theme: PropTypes.oneOf(["primary", "secondary"]),
-  };
-
-  static defaultProps = {
-    currentPage: "home",
-    theme: "primary",
-  };
-
-  get colors() {
-    const { theme } = this.props;
+    const COLOR_THEME_SECONDARY = {
+      color: "baby-blue",
+      underlineColor: "baby-blue",
+      logoColor: "white",
+    };
 
     return theme === "primary" ? COLOR_THEME_PRIMARY : COLOR_THEME_SECONDARY;
-  }
+  };
 
-  renderBrand = () => {
-    const { logoColor } = this.colors;
+  const { color, underlineColor, logoColor } = colors();
+  const [open, openMenu] = useState(false);
```

Comment:
I would use [menuOpen, setMenuOpen] otherwise if you just see `open` you don't know what it refers

Reply:
Yes, it's better 👍